contracts/mock/util/SignatureValidator.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+contract SignatureValidator {
```

Comment:
We should have this contract extend the consensus contract to get all the functionality directly from them

Reply:
Fixed